### DISCUSSION TITLE
Game Features

### DISCUSSION THREAD

**Discussion by steve-langmers** [davidmerwin/LangMersSystems#9] [Ideas]
LangMers™

**Comment by steve-langmers** [comment]
@davidmerwin @steve-langmers  <img src="https://cdn.mathpix.com/snip/images/uDg3MXr8olzip8oWRZM9X4hlYFmjJhTamiiF_bvzpKk.original.fullsize.png" /> #8 #7 #2 #4 #6

**Comment by davidmerwin** [comment]
> @davidmerwin @steve-langmers  <img src="https://cdn.mathpix.com/snip/images/uDg3MXr8olzip8oWRZM9X4hlYFmjJhTamiiF_bvzpKk.original.fullsize.png" /> #8 #7 #2 #4 #6  

@steve-langmers @davidmerwin

**Comment by davidmerwin** [comment]
> @davidmerwin @steve-langmers  <img src="https://cdn.mathpix.com/snip/images/uDg3MXr8olzip8oWRZM9X4hlYFmjJhTamiiF_bvzpKk.original.fullsize.png" /> #8 #7 #2 #4 #6  

@steve-langmers @davidmerwin ![image](https://github.com/davidmerwin/LangMersSystems/assets/52674935/c9ec472c-d5f1-42f2-83f0-d19f5c582557)

**Comment by davidmerwin** [comment]
> LangMers™ 

![image](https://github.com/davidmerwin/LangMersSystems/assets/52674935/8236ed88-c9da-4060-baa9-94bba5296f96) @steve-langmers @davidmerwin ![image](https://github.com/davidmerwin/LangMersSystems/assets/52674935/e0057208-60a5-4e35-b40d-f6787887d5fc)
